src/math/tfm_desc.c

```diff
@@ -741,6 +776,7 @@ const ltc_math_descriptor tfm_desc = {
    &mul,
    &muli,
    &sqr,
+   &sqrtmod_prime,
```

Comment:
why can't we just set this to NULL and check before the invocation?
Just an idea...
AFAIU there's no functional difference of compressed and uncompressed points, right?
so we could disable compression at runtime in case tfm is used... (in fact that would be the same behavior at runtime than what is currently done in `ecc_encrypt_key()` at compile-time)

Reply:
done